src/lib/config-utils.js

```diff
@@ -21,27 +21,26 @@ const BLACKLIST = {
 };
 
 /**
- * Normalise user list.
+ * Normalize user list.
  * @return {Array}
  */
-export function normalizeUserlist(oldFormat: any, newFormat: any) {
+export function normalizeUserList(...args: any): Array<string> {
```

Comment:
`normalizeUserList` to be sure this works fine. A unit test is required.  I'd do it undoing this changes and test both implementations.